src/feature/settings/schemas/aiSettingsSchema.ts

```diff
@@ -0,0 +1,19 @@
+import z from "zod";
+
+export const aiSettingsSchema = z.object({
+	aiApiKey: z.string(),
+	aiModel: z.enum(["chatgpt", "mistral", "deepseek"]),
+});
+
+export type AiSettingsSchema = z.infer<typeof aiSettingsSchema>;
+
+export const aiContextSchema = z.object({
+	companyActivity: z.string().optional(),
+	companyExperience: z.string().optional(),
+	companyIndustry: z.string().optional(),
+	companyName: z.string().optional(),
```

Comment:
these shouldn't be optional 
```suggestion
	companyActivity: z.string(),
	companyExperience: z.string(),
	companyIndustry: z.string(),
	companyName: z.string(),
```